appeals/web/src/server/lib/mappers/__tests__/mappers.test.js

```diff
@@ -146,6 +147,42 @@ describe('pagination mapper', () => {
 				`${testBaseUrl}?pageSize=10&pageNumber=5${testAdditionalQueryString}`
 			);
 		});
+
+		it('renders an ellipsis element before the final page marker and no ellipsis element after the first page marker when page count is greater than 10 and current page is 2', () => {
+			const testBaseUrl = 'test-base-url';
+			const result = mapPagination(2, 15, 30, testBaseUrl, testAdditionalQuery);
+
+			const containsEllipsisElementAfterFirstPageMarker = isEqual(result.items[1], {
+				ellipsis: true
+			});
+			const containsEllipsisElementBeforeLastPageMarker = isEqual(result.items[3], {
+				ellipsis: true
+			});
+			const containsBlankEllipsisElementBeforeLastPageMarker =
```

Comment:
Very minor, this should probably be renamed to just `containsBlankEllipsisElement`